cockatrice/src/dlg_register.cpp

```diff
@@ -62,9 +62,7 @@ DlgRegister::DlgRegister(QWidget *parent)
     countryEdit->setCurrentIndex(0);
     QStringList countries = settingsCache->getCountries();
     foreach(QString c, countries)
-    {
-        countryEdit->addItem(QPixmap(":/resources/countries/" + c + ".svg"), c);
-    }
+        countryEdit->addItem(QPixmap(":/resources/countries/" + c.toLower() + ".svg"), c);
```

Comment:
The issue that was experienced is the characters in the DB were being
written first character in upper. But I couldn't reproduce so I have no
idea how it was happening.

On Tuesday, August 18, 2015, ctrlaltca <notifications@github.com> wrote:

> In cockatrice/src/dlg_register.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1381#discussion_r37349356>:
>
> > @@ -62,9 +62,7 @@ DlgRegister::DlgRegister(QWidget *parent)
> >      countryEdit->setCurrentIndex(0);
> >      QStringList countries = settingsCache->getCountries();
> >      foreach(QString c, countries)
> > -    {
> > -        countryEdit->addItem(QPixmap(":/resources/countries/" + c + ".svg"), c);
> > -    }
> > +        countryEdit->addItem(QPixmap(":/resources/countries/" + c.toLower() + ".svg"), c);
>
> Same as before: c values are hardcoded lowercase, no need to lower them
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1381/files#r37349356>.
>